content/engine/api/sdk/_index.md

```diff
@@ -81,54 +81,54 @@ import (
 	"io"
 	"os"
 
-	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/container"
-        "github.com/docker/docker/api/types/image"
+	"github.com/docker/docker/api/types/image"
 	"github.com/docker/docker/client"
 	"github.com/docker/docker/pkg/stdcopy"
 )
 
 func main() {
-    ctx := context.Background()
-    cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
-    if err != nil {
-        panic(err)
-    }
-    defer cli.Close()
-
-    reader, err := cli.ImagePull(ctx, "docker.io/library/alpine", image.PullOptions{})
-    if err != nil {
-        panic(err)
-    }
-    io.Copy(os.Stdout, reader)
-
-    resp, err := cli.ContainerCreate(ctx, &container.Config{
-        Image: "alpine",
-        Cmd:   []string{"echo", "hello world"},
-    }, nil, nil, nil, "")
-    if err != nil {
-        panic(err)
-    }
-
-    if err := cli.ContainerStart(ctx, resp.ID, types.ContainerStartOptions{}); err != nil {
-        panic(err)
-    }
-
-    statusCh, errCh := cli.ContainerWait(ctx, resp.ID, container.WaitConditionNotRunning)
-    select {
-    case err := <-errCh:
-        if err != nil {
-            panic(err)
-        }
-    case <-statusCh:
-    }
-
-    out, err := cli.ContainerLogs(ctx, resp.ID, types.ContainerLogsOptions{ShowStdout: true})
-    if err != nil {
-        panic(err)
-    }
-
-    stdcopy.StdCopy(os.Stdout, os.Stderr, out)
+	ctx := context.Background()
```

Comment:
For others reviewing; the large number of lines changes is due to spaces -> tabs. Go uses tabs for indenting, but let me double-check with @dvdksn if there perhaps was a reason to use spaces for the docs (maybe for formatting reasons)?

To review the changes with whitespace changes hidden, add `?w=1` to the URL; https://github.com/docker/docs/pull/20530/files?w=1